packages/suite/src/views/onboarding/steps/Firmware/index.tsx

```diff
@@ -57,7 +57,7 @@ const FirmwareStep = () => {
                 image="FIRMWARE"
                 heading={<Translation id="TR_FW_INSTALLATION_FAILED" />}
                 description={<Translation id="TOAST_GENERIC_ERROR" values={{ error }} />}
-                innerActions={<RetryButton onClick={firmwareUpdate} />}
+                innerActions={<RetryButton onClick={() => firmwareUpdate()} />}
```

Comment:
Why changing this?

Reply:
Because `firmwareUpdate` now receives an argument which is not the default event, so TypeScript complains:
_Type 'MouseEvent<HTMLButtonElement, MouseEvent>' is not assignable to type 'FirmwareType | undefined'_